opennms-webapp/src/main/webapp/WEB-INF/applicationContext-spring-security.xml

```diff
@@ -98,45 +98,48 @@
   <http pattern="/webstart/**" security="none"/>
 
   <!-- Only one <http> section can match the implicit '/**' pattern -->
-  <http pattern="/**" access-denied-page="/accessDenied.jsp" realm="OpenNMS Realm" auto-config="false" entry-point-ref="loginUrlAuthenticationEntryPoint">
-    <intercept-url pattern="/" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/favicon.ico" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/logoff.jsp" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/login.jsp*" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/accessDenied.jsp*" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/css/**" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/svg/**" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/js/global.js" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/lib/**" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/images/**" access="ROLE_ANONYMOUS,ROLE_USER,ROLE_DASHBOARD" />
-    <intercept-url pattern="/admin/ng-requisitions/**" access="ROLE_PROVISION,ROLE_ADMIN" />
-    <intercept-url pattern="/rest/snmpConfig/**" access="ROLE_PROVISION,ROLE_ADMIN" />
-    <intercept-url pattern="/admin/**" access="ROLE_ADMIN" />
-    <intercept-url pattern="/**/mib-compiler" access="ROLE_ADMIN" />
-    <intercept-url pattern="/**/admin-*" access="ROLE_ADMIN" />
-    <intercept-url pattern="/rtc/post/**" access="ROLE_RTC" />
-    <intercept-url pattern="/element/rescan.jsp" access="ROLE_ADMIN" />
+  <http pattern="/**" use-expressions="true" access-denied-page="/accessDenied.jsp" realm="OpenNMS Realm" auto-config="false" entry-point-ref="loginUrlAuthenticationEntryPoint">
```

Comment:
Is the switch to using expressions necessary for disabling access to `/osgi/**`?